index.js

```diff
@@ -407,9 +406,7 @@ Router.prototype.findRoute = function findNode (method, path, constraints = {})
       // add the wildcard parameter
       params.push('*')
       currentNode = currentNode.getWildcardChild()
-      if (currentNode === null) {
```

Comment:
Uncovered branch.

Can't find a good reason why this could be null using apis.